web-common/src/lib/number-formatting/strategies/per-range-default-options.ts

```diff
@@ -4,6 +4,45 @@ import {
   NumberKind,
 } from "../humanizer-types";
 
+export const defaultNoFormattingOptions: FormatterOptionsCommon &
```

Comment:
This object doesn't strike me as "options", but rather a selected configuration.